script/package.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -o errexit
+set -o noclobber
+set -o nounset
+set -o pipefail
+
+if [ -z "$1" ]
+  then
+    echo "Usage: $0 package_root [ compress_type ]"
+    echo "compress_type must be one of the types accepted by dpkg-deb (-Z), default to 'xz'"
+    exit 1
+fi
+
+readonly package_root="$1"
+readonly compress_type="${2:-xz}"
+
+# calculate uncompressed size in kB, rounded up
+package_root_size="$(du --bytes --summarize --exclude=DEBIAN "${package_root}"/ | awk '{print $1}')"
+readonly package_root_size
+readonly installed_size="$(( (package_root_size + 1024 - 1) / 1024 ))"
+
+# # creates control file
+mkdir -p "${package_root}/DEBIAN/"
+cat << EOF | envsubst > "${package_root}/DEBIAN/control"
+Package: ${PACKAGE}
+Version: ${VERSION}
+Installed-Size: ${installed_size}
+Architecture: ${ARCH}
+Maintainer: ${MAINTAINER}
+Depends: ${DEPENDS}
+Description: ${DESCRIPTION}
+Homepage: https://github.com/charlie-foxtrot/RTLSDR-Airband.git
+EOF
+
+# post install script to symlink default config
+cat << EOF > "${package_root}/DEBIAN/postinst"
+#!/bin/sh
+
+set -o errexit
+
+if [ ! -e /usr/share/rtl_airband/config/default.conf ]; then
+  ln --symbolic /usr/share/rtl_airband/config/basic_emergency.conf /usr/share/rtl_airband/config/default.conf
```

Comment:
if a config file is not provided at the command line then `rtl_airband` will default to looking for `/usr/local/etc/rtl_airband.conf`, should the symlink be placed there?  NOTE: currently `rtl_airband.service` does not provide a config path so would be looking for `/usr/local/etc/rtl_airband.conf`